tests/dummy/app/templates/signup-example.hbs

```diff
@@ -1,20 +1,20 @@
 <h1>Signup example</h1>
 
-<form {{action "signup" on="submit"}}>
+<form class="signup-example-form" {{action "signup" on="submit"}}>
```

Comment:
We have an example form component used for component tests. Would he be feasible to combine the two use cases, e.g. test a form component via integration test and use the same form component in the app for an acceptance test?